### ISSUE TITLE
[Change Security Question - remove] button doesn't always work

### ISSUE THREAD

**Issue by andrewleponis** [AscentMS/NHSD#93]
1. Click :- [Change Security Question]
2. Add a new security question and click save
3. Click :- [Change Security Question]
4. Click:- [Remove] only once on for the security question you have just added.   

Result: - Question is not removed (Click again and the Question is removed)

**Comment by Saf456** [comment]
Yes replicated - low priority for the pilot

**Comment by Saf456** [comment]
tested  - correct question removed
